plugins.txt

```diff
@@ -0,0 +1,13 @@
+https://nc.igorek.dev/s/K7XMYQS9DJk2EF7/download?path=%2F&files=AuthMe.jar@AuthMe.jar
```

Comment:
just a quick question: should we updates the plugins links if we will update them on the nextcloud?

Reply:
Yes!

Reply:
ok, i got it